draft-ietf-ppm-dap.md

```diff
@@ -1377,21 +1383,26 @@ This message consists of:
 * `prepare_inits`: the sequence of `PrepareInit` messages constructed in the
   previous step.
 
-Finally, the Leader sends a PUT request to
-`{helper}/tasks/{task-id}/aggregation_jobs/{aggregation-job-id}`. The payload
-is set to `AggregationJobInitReq` and the media type is set to
+Finally, the Leader sends a request to
+`PUT {helper}/tasks/{task-id}/aggregation_jobs/{aggregation-job-id}`. The
```

Comment:
nit: I don't love this phrasing. `PUT` is the verb in the request sent to the path, not the recipient of the request.

Reply:
I mostly took this to be consistent with the existing phrasing of https://www.ietf.org/archive/id/draft-ietf-ppm-dap-09.html#section-4.6.1-14, but now perusing the draft it looks like that was in fact an outlier usage of `PUT /blah`. I will rephrase as `sends a PUT request to "/blah"`.

Reply:
IMO consistency is king here, but also it's not worth sweating this too hard, because I suspect that things like request paths might get revisited as DAP makes its way to RFC (specifically I suspect we'll get forced to do an ACME style directory).